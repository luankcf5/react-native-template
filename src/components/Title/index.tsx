import styled from "styled-components/native";

const Title = styled.Text`
  font-size: 24px;
  color: ${({ theme }) => theme.colors.title};
  margin-top: 120px;
`;

export default Title;
