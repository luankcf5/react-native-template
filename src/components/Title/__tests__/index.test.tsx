import * as React from "react";
import * as renderer from "react-test-renderer";
import WithTheme from "theme/withTheme";
import Title from "..";

describe("Title component UI", () => {
  test("Should render properly", () => {
    const tree = renderer
      .create(
        <WithTheme>
          <Title>Fastics Native Title</Title>
        </WithTheme>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
