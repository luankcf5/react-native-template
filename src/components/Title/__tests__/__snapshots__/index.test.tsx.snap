// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component UI Should render properly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "black",
        "fontSize": 24,
        "marginTop": 120,
      },
    ]
  }
>
  Fastics Native Title
</Text>
`;
